dm/syncer/ddl_test.go

```diff
@@ -789,6 +789,66 @@ func TestAdjustCollation(t *testing.T) {
 	}
 }
 
+func TestAdjustAutoIDCache(t *testing.T) {
+	cases := []struct {
+		sql         string
+		expectedSQL string
+		cacheSize   uint64
+	}{
+		{
+			"CREATE TABLE `test`.`t1` (`id` INT)",
+			"CREATE TABLE `test`.`t1` (`id` INT)",
+			0,
+		},
+		{
+			"CREATE TABLE `test`.`t1` (`id` INT AUTO_INCREMENT)",
+			"CREATE TABLE `test`.`t1` (`id` INT AUTO_INCREMENT)",
+			0,
+		},
+		{
+			"CREATE TABLE `test`.`t1` (`id` INT AUTO_INCREMENT)",
+			"CREATE TABLE `test`.`t1` (`id` INT AUTO_INCREMENT) /*T![auto_id_cache] AUTO_ID_CACHE = 1 */",
+			1,
+		},
+		{
+			"CREATE TABLE `test`.`t1` (`id` INT AUTO_INCREMENT)",
+			"CREATE TABLE `test`.`t1` (`id` INT AUTO_INCREMENT) /*T![auto_id_cache] AUTO_ID_CACHE = 30 */",
+			30,
+		},
+	}
```

Comment:
seems you didn't cover cases that original CREATE TABLE already set auto_id_cache

Reply:
thanks, added a relevant test case